test/Driver/response-files-with-spaces-in-filenames.swift

```diff
@@ -0,0 +1,4 @@
+// RUN: %empty-directory(%t)
+// RUN: touch "%t/f i l e.swift"
+//
+// RUN: %swiftc_driver -driver-force-response-files -typecheck  "%t/f i l e.swift"
```

Comment:
This needs to be `%target-build-swift`, since it's going to actually run the frontend. I suppose you could use `-parse` instead of `-typecheck` and then it doesn't need the standard library or anything.

Reply:
Thanks! I've made those changes.